plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/TestS3WrongRegionPicked.java

```diff
@@ -27,12 +27,14 @@
     public void testS3WrongRegionSelection()
             throws Exception
     {
-        try (HiveMinioDataLake dataLake = new HiveMinioDataLake("test-bucket").start();
+        String bucketName = "test-bucket" + randomTableSuffix();
```

Comment:
Can we add a comment on why we need this random bucket name ?

Reply:
`MinioClient` has something which is called `createdBuckets` which is static. Event if we start new Minio container, this will fail to create bucket with same name.

Reply:
But in the new minio container, the bucket doesn't exist right ?

Reply:
In `MinioClient` class from package `io.trino.testing.minio`.

Reply:
This `MinioClient` class is strictly for testing purposes, so forcing a unique bucket naming seems like a good idea. If I had thought about it in the first place, there would be no bug.

Reply:
I would look up on `BackgroundHiveSplitLoader` if it behaves correctly when stopped and `loadSplits` is ongoing.